Cargo.toml

```diff
@@ -8,11 +8,11 @@ license-file = "LICENSE.txt"
 
 [workspace.dependencies]
 derive_builder = "0.20"
-derive_more = { version = "1.0", features = ["constructor", "display", "from", "into", "debug"] }
+derive_more = { version = "2.0", features = ["constructor", "display", "from", "into", "debug"] }
 thiserror = "2"
 tonic = "0.12"
 tonic-build = "0.12"
-opentelemetry = { version = "0.26", features = ["metrics"] }
+opentelemetry = { version = "0.28", features = ["metrics"] }
```

Comment:
So the options are: 1) don't upgrade `opentelemetry` and therefore stay on old `opentelemetry-prometheus`, 2) upgrade `opentelemetry` and start getting a lot of info-level log spam (assuming langs have enabled info level which I don't think they do by default but will by annoying for users that do)

Tough choice, but assuming https://github.com/open-telemetry/opentelemetry-rust/issues/2697 moves quickly and 0.29 is not too far off, maybe option 1 is best for a bit longer.